README.md

```diff
@@ -18,20 +18,23 @@ are welcome.
    [io.js](https://iojs.org/en/index.html) might work, too, although
    it is currently not tested.
 2. Clone the repo, install needed npm packages:
-   
     ```
     git clone https://github.com/metacran/metacranweb
-	cd metacranweb
-	npm install
-	```
-
-3. Start the app:
-
+	  cd metacranweb
+	  npm install
+	  ```
+3. Start a local [redis](https://redis.io/) instance, for example with Docker:
     ```
-    PORT=3000 bin/www
-	```
+    docker run --name metacran-redis -p 6379:6379 redis redis-server
+    ```
+    Alternatively you can configure an exising redis server with the environment variables
+    `REDIS_HOST` (default: `'127.0.0.1'`) and `REDIS_PORT` (default: `6379`)
+4. Start the app:
 
-4. Go to `http://localhost:3000` in your browser.
+    ```Ready check failed: Redis connection gone from end event.
```

Comment:
Where is this message coming from?

Reply:
That looks like a copy and paste mistake during me figuring out how to run Redis, will remove it.